remo/profiles/tasks.py

```diff
@@ -53,6 +53,9 @@ def check_mozillian_username():
             user.first_name = 'Anonymous'
             user.last_name = 'Mozillian'
             user.userprofile.mozillian_username = ''
+        if len(user.last_name) > 30 or len(user.first_name) > 30:
```

Comment:
This way when ``len(user.last_name) > 30`` we are also slicing ``first_name``. Please fix the logic here.